src/definitions/behaviors/form.js

```diff
@@ -789,8 +787,8 @@ $.fn.form = function(parameters) {
           },
           // alias
           rules: function(field, rules) {
-            if(Array.isArray(field)) {
-              $.each(fields, function(index, field) {
+            if (Array.isArray(field)) {
+              $.each(field, function(index, field) {
```

Comment:
This issue was found by [`no-undef` rule](https://eslint.org/docs/rules/no-undef)

There is a separate PR #556 to fix this.